src/Detail/Blitline/Job/Definition/JobDefinition.php

```diff
@@ -2,62 +2,95 @@
 
 namespace Detail\Blitline\Job\Definition;
 
-use Detail\Blitline\Job\Source;
-
 class JobDefinition extends BaseDefinition implements JobDefinitionInterface
 {
     const OPTION_SOURCE       = 'src';
     const OPTION_POSTBACK_URL = 'postback_url';
     const OPTION_VERSION      = 'v';
     const OPTION_FUNCTIONS    = 'functions';
+    const OPTION_SOURCE_TYPE  = 'src_type';
+    const OPTION_SOURCE_DATA  = 'src_data';
 
     /**
      * @var array
      */
     protected $options = array(
-        self::OPTION_VERSION   => '1.21',
+        self::OPTION_VERSION   => '1.22',
```

Comment:
The documentation says 1.21 is the latest (see http://docs.blitline.com/articles/jobs.html). Is 1.22 correct?